### PR TITLE
CEPHSTORA-91 Rework deploy-ceph.yml to only deploy ceph.

### PR DESCRIPTION
**Pull request by andymcc**
This PR moves the additional bits out of deploy-ceph and leaves them as
individual playbooks. That ensures we only setup Ceph, and any
additional bits (logging, keystone integration etc.) are all only done
when called explicitly.

The documentation has been updated to reflect this.

### CODE REVIEW

**Comment by andymcc** [comment]
recheck_cit_xenial_functional_test

File: docs/rgw_setup.md [hunk 1]
```diff
@@ -1,22 +1,19 @@
...
 ## Role of rpc-ceph
-The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and versioned software for RPC's use.  
+The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and software for RPC's use.
 
 ### List of Use Cases
 
-* General Installation using ceph-ansible without integration to an Auth System
-* Configuring rgw to use an instance of OpenStack Keystone
+* General Installation using ceph-ansible **without** preconfigured external Auth System
+* General Installation using ceph-ansible **with** preconfigured external Auth System
```
Comment (prometheanfire) [inline docs/rgw_setup.md]: a preconfigured

File: docs/rgw_setup.md [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 
 ## Role of rpc-ceph
-The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and versioned software for RPC's use.  
+The purpose of (rpc-ceph)[https://github.com/rcbops/rpc-ceph] is to provide a consistant versioned set of processes, variables and software for RPC's use.
 
 ### List of Use Cases
 
-* General Installation using ceph-ansible without integration to an Auth System
-* Configuring rgw to use an instance of OpenStack Keystone
+* General Installation using ceph-ansible **without** preconfigured external Auth System
```
Comment (prometheanfire) [inline docs/rgw_setup.md]: a preconfigured

File: docs/rgw_setup.md [hunk 3]
```diff
@@ -1,22 +1,19 @@
 # Setting up Ceph Object Gateway for different RPC Use Cases
 
-Ceph Object Gateway(rgw) is an object storage interface built on top of librados to provide applications with a RESTful gateway to Ceph Storage Clusters. Ceph Object Storage supports two interfaces  S3 and OpenStack Swift.   
+Ceph Object Gateway(rgw) is an object storage interface built on top of librados to provide applications with a RESTful gateway to Ceph Storage Clusters. Ceph Object Storage supports two interfaces  S3 and OpenStack Swift.
```
Comment (prometheanfire) [inline docs/rgw_setup.md]: extra spaace before S3

File: docs/rgw_setup.md [hunk 4]
```diff
@@ -26,68 +23,43 @@ ceph-rgw01
...
+* radosgw_keystone ***(set to True)**
 * radosgw_keystone_admin_user
 * radosgw_keystone_admin_password
 * radosgw_keystone_admin_tenant
 
-### Run the deployment
-```bash
-ceph-ansible -i <link to your inventory file> playbooks/deploy-ceph.yml -e@<link to your vars file>
-```
+**NB** If your Auth system is preconfigured user, tenant & password settings need to match the Auth system's settings.
```
Comment (prometheanfire) [inline docs/rgw_setup.md]: If your Auth system is preconfigured: user, tenant & password settings need to match the Auth system's settings.

**Comment by andymcc** [comment]
recheck_cit_xenial_keystone_rgw_test

**Pull request merged**